### ISSUE TITLE
Installation on Linux by following the guide step by step fails with UnsatisfiableError 

### ISSUE THREAD

**Issue by DrSchteeven** [facebookresearch/pytorch3d#1022]
## 🐛 Bugs / Unexpected behaviors
I've tried to install pytorch3d by following the installation instructions step by step, I'm pretty sure I've gone through each command, but at the last step of ```conda install pytorch3d -c pytorch3d``` the installation fails with UnsatisfiableError and I'm trying to understand what's wrong.

## Instructions To Reproduce the Issue:
1. The exact command(s) you ran: I followed the installation guide of pytorch3d step by step, creating a new environment for it, I'm attaching a file with the full terminal session (I've redacted the username and pc name).
2. What you observed (including the full logs):

Here is the beginning of the error, the rest is in the attachment:
```
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - colorama -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - olefile -> python[version='>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']
  - torchaudio -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']
  - torchvision -> python[version='>=3.10,<3.11.0a0']
  - wheel -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']

Your python: python=3.9

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.

The following specifications were found to be incompatible with a past
explicit spec that is not an explicit spec in this operation (torchvision):

  - pytorch3d -> cudatoolkit[version='>=10.1,<10.2|>=11.0,<11.1|>=11.1,<11.2|>=11.3,<11.4|>=10.2,<10.3|>=9.2,<9.3']
  - pytorch3d -> fvcore
  - pytorch3d -> iopath
  - pytorch3d -> pytorch==1.4 -> cudatoolkit[version='>=10.0.130,<10.1.0a0|>=10.1.243,<10.2.0a0|>=9.2,<9.3.0a0']
  - pytorch3d -> pytorch[version='1.10.0|1.9.1|1.8.1|1.9.0|1.8.0|1.7.1|1.7.0|1.6.0|1.5.1|1.5.0|1.4']
  - pytorch3d -> torchvision[version='>=0.5'] -> pytorch[version='1.10.1|1.7.1.*']
```

Here is also the output of  ```conda list```

```
(pytorch3d) user@computer:/$ conda list
# packages in environment at /home/user/miniconda3/envs/pytorch3d:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main
_openmp_mutex             4.5                       1_gnu
blas                      1.0                         mkl
bzip2                     1.0.8                h7b6447c_0
ca-certificates           2021.10.26           h06a4308_2
certifi                   2021.10.8        py39hf3d152e_1    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
cudatoolkit               11.3.1               h2bc3f7f_2
ffmpeg                    4.3                  hf484d3e_0    pytorch
freetype                  2.11.0               h70c0345_0
fvcore                    0.1.5.post20210804    pyhd8ed1ab_0    conda-forge
giflib                    5.2.1                h7b6447c_0
gmp                       6.2.1                h2531618_2
gnutls                    3.6.15               he1e5248_0
intel-openmp              2021.4.0          h06a4308_3561
iopath                    0.1.9                      py39    iopath
jpeg                      9d                   h7f8727e_0
lame                      3.100                h7b6447c_0
lcms2                     2.12                 h3be6417_0
ld_impl_linux-64          2.35.1               h7274673_9
libffi                    3.3                  he6710b0_2
libgcc-ng                 9.3.0               h5101ec6_17
libgomp                   9.3.0               h5101ec6_17
libiconv                  1.15                 h63c8f33_5
libidn2                   2.3.2                h7f8727e_0
libpng                    1.6.37               hbc83047_0
libstdcxx-ng              9.3.0               hd4cf53a_17
libtasn1                  4.16.0               h27cfd23_0
libtiff                   4.2.0                h85742a9_0
libunistring              0.9.10               h27cfd23_0
libuv                     1.40.0               h7b6447c_0
libwebp                   1.2.0                h89dd481_0
libwebp-base              1.2.0                h27cfd23_0
lz4-c                     1.9.3                h295c915_1
mkl                       2021.4.0           h06a4308_640
mkl-service               2.4.0            py39h7f8727e_0
mkl_fft                   1.3.1            py39hd3c417c_0
mkl_random                1.2.2            py39h51133e4_0
ncurses                   6.3                  h7f8727e_2
nettle                    3.7.3                hbbd107a_1
ninja                     1.10.2           py39hd09550d_3
numpy                     1.21.2           py39h20f2e39_0
numpy-base                1.21.2           py39h79a1101_0
nvidiacub                 1.10.0                        0    bottler
olefile                   0.46               pyhd3eb1b0_0
openh264                  2.1.1                h4ff587b_0
openssl                   1.1.1l               h7f8727e_0
pillow                    8.4.0            py39h5aabda8_0
pip                       21.2.4           py39h06a4308_0
portalocker               2.3.2            py39hf3d152e_1    conda-forge
python                    3.9.7                h12debd9_1
python_abi                3.9                      2_cp39    conda-forge
pytorch                   1.10.1          py3.9_cuda11.3_cudnn8.2.0_0    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pyyaml                    5.4.1            py39h3811e60_0    conda-forge
readline                  8.1                  h27cfd23_0
setuptools                58.0.4           py39h06a4308_0
six                       1.16.0             pyhd3eb1b0_0
sqlite                    3.37.0               hc218d9a_0
tabulate                  0.8.9              pyhd8ed1ab_0    conda-forge
termcolor                 1.1.0                      py_2    conda-forge
tk                        8.6.11               h1ccaba5_0
torchaudio                0.10.1               py39_cu113    pytorch
torchvision               0.11.2               py39_cu113    pytorch
tqdm                      4.62.3             pyhd8ed1ab_0    conda-forge
typing_extensions         3.10.0.2           pyh06a4308_0
tzdata                    2021e                hda174b7_0
wheel                     0.37.0             pyhd3eb1b0_1
xz                        5.2.5                h7b6447c_0
yacs                      0.1.6                      py_0    conda-forge
yaml                      0.2.5                h516909a_0    conda-forge
zlib                      1.2.11               h7f8727e_4
zstd                      1.4.9                haebb681_0
```

After the first UnsatisfiableError I've tried installing a later version of pytorch according to their Get Started page with: ```conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch``` but it didn't solve the issue.

In case it might be useful, I'm running this command inside an Ubuntu 20.04.03 inside WLS2 with Linux 5.10.60.1-microsoft-standard-WSL2 #1 SMP Wed Aug 25 23:20:18 UTC 2021 x86_64 x86_64 x86_64 and NVIDIA-SMI 495.53 Driver Version: 497.29 CUDA Version: 11.5

Thanks in advance for any help

[pytorch3d_terminaloutput.txt](https://github.com/facebookresearch/pytorch3d/files/7824346/pytorch3d_terminaloutput.txt)

**Comment by bottler** [comment]
PyTorch 1.10.1 was released around the same time as the latest PyTorch3D release 0.6.1 (15-16 December 2021), and the PyTorch3D release did not include conda builds for it. There are builds for 1.10.0 and several earlier versions. If you start a new environment and fix the pytorch version to 1.10.0 then the same commands should work.

I plan to add a nightly conda build for PyTorch 1.10.1 soon, but even that does not exist quite yet.

There is in fact a wheel build of PyTorch3D 0.6.1 with PyTorch 1.10.1. You may be able to get it in your current environment with 
`pip install --no-cache --no-index pytorch3d -f https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py39_cu113_pyt1101/download.html`.

**Comment by bottler** [comment]
Nightly install will now work.
